pkg/controller/cronjob/utils_test.go

```diff
@@ -87,17 +87,23 @@ func TestGetJobFromTemplate(t *testing.T) {
 	if len(job.ObjectMeta.Annotations) != 2 {
 		t.Errorf("Wrong number of annotations")
 	}
-	v, ok := job.ObjectMeta.Annotations[v1.CreatedByAnnotation]
+	_, ok := job.ObjectMeta.Annotations[v1.CreatedByAnnotation]
```

Comment:
no, i haven't started working on cronjob at that moment. :)

i removed `v` as i thought annotation testing is unnecessary. after second thought, the annotation testing should still be there. although we do not depend on the annotation anymore, but we still have it at `getJobFromTemplate()`. it is still better to test the annotation.